hydra/functions/FlatteLineShape.h

```diff
@@ -223,19 +223,19 @@ namespace hydra {
                     rhopipi_imag = (2. / 3) * ::sqrt(-1 + twopimasssq / s);
```

Comment:
Please, try to eliminate the ```if/else```  instructions using boolean arithmetic. Avoid ```value.```, enforce ```value.0```.